ComfortZoneApp/app/src/main/java/com/example/comfortzone/ui/HostActivity.java

```diff
@@ -125,7 +125,7 @@ private void listenerSetup() {
         bottomNavigationView.setOnItemSelectedListener(item -> {
             Pair<Integer, Integer> animations;
             if (isLoading) {
-                animations = setAnimationLeftToRight();
+                animations = new Pair<>(0,0);
```

Comment:
hmm what does resource id 0 mean? can you check instead if animations is null when setting custom animations?

Reply:
this means no animations